schema/method_units/mathematical/fapprx/basisexp.json

```diff
@@ -23,6 +23,29 @@
                     "slug": "none"
                 }
             ]
+        },
+        "metadata": {
+            "$ref": "#/definitions/metadata"
+        }
+    },
+    "definitions": {
+        "metadata": {
+            "allOf": [
+                {
+                    "$ref": "../fapprx.json#/definitions/metadata-target-function"
+                },
+                {
+                    "properties": {
+                        "basisFunctionType": {
+                            "description": "Name or class of basis functions",
+                            "type": "string"
+                        }
+                    },
+                    "required": [
+                        "basisFunctionType"
+                    ]
+                }
+            ]
```

Comment:
The above implementation seems a bit convoluted, do we need to reuse the whole `metadata` field or just expose the `basisFunctionType` at the top level?